th2_data_services_lwdp/commands/http.py

```diff
@@ -271,8 +277,8 @@ def _sse_bytes_stream(self, data_source: HTTPDataSource):  # noqa
         api: HTTPAPI = data_source.source_api
         url = api.get_url_get_pages_info(
             self._book_id,
-            _datetime2ms(self._start_timestamp),
-            _datetime2ms(self._end_timestamp),
+            _check_millisecond(_datetime2ms(self._start_timestamp)),
+            _check_millisecond(_datetime2ms(self._end_timestamp)),
```

Comment:
That will work wrong
Think what's wrong here.

Reply:
By wrong do you mean check should happen during initialization and not after?
Check moved to initialization of class.

Reply:
No, the problem is not related with it.
The problem in `what you do` not in `where you do it`